R/opts.R

```diff
@@ -455,21 +462,30 @@ backcalc_opts <- function(prior = c("reports", "none", "infections"),
 #'
 #' # add a custom length scale
 #' gp_opts(ls_mean = 4)
+#'
+#' # use linear kernel
+#' gp_opts(kernel = "periodic")
 gp_opts <- function(basis_prop = 0.2,
                     boundary_scale = 1.5,
                     ls_mean = 21,
                     ls_sd = 7,
                     ls_min = 0,
                     ls_max = 60,
-                    alpha_sd = 0.05,
-                    kernel = c("matern", "se", "ou"),
+                    alpha_mean = 0,
+                    alpha_sd = 0.01,
+                    kernel = c("matern", "se", "ou", "periodic"),
                     matern_order = 3 / 2,
```

Comment:
Making the options explicit.
```suggestion
                    matern_order = c(3 / 2, 1 / 2, 5 /2),
```

Reply:
As its numeric I don't think that will work. It also isn't limited to those options